deploy-manage/autoscaling/ec-hosted-autoscaling-api-example.md

```diff
@@ -1,14 +1,16 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud/current/ec-autoscaling-api-example.html
+applies_to:
+  deployment:
+    ess: ga
 ---
 
-# Autoscaling through the API [ec-autoscaling-api-example]
+# {{ech}} autoscaling through the API [ec-autoscaling-api-example]
```

Comment:
IMO, the two API pages could be merged (and also included on a specific ECE/ECH page).

The only thing that is significantly different is the API request example - we can use tabs for that.